draft-ietf-taps-transport-security.md

```diff
@@ -156,7 +157,10 @@ Datagram Transport Layer Security (DTLS), QUIC + TLS, tcpcrypt, Internet Key Exc
 with Encapsulating Security Protocol (IKEv2 + ESP), SRTP (with DTLS), WireGuard, CurveCP,
 and MinimalT. For each protocol, this document provides a brief description, the security features it
 provides, and the dependencies it has on the underlying transport. This is followed by defining the
-set of transport security features shared by these protocols. Finally, the document distills the application and
+set of transport security features shared by these protocols. The document groups these security features
+into a minimal set of mandatory features, which every secure transport system should provide in addition to
```

Comment:
Hmm, it's strange to say that features are mandatory and then later say "every secure transport system *should* provide..." Maybe we should just say:

"The document groups these security features into a minimal set of features, which every secure transport system should provide in addition to the transport features described in {{?I-D.ietf-taps-minset}}, and additional optional features, which may not be available in every secure transport system."

Reply:
Sure, that works, too. Changed it.
Section 5.1 is named "Mandatory features" even though this is an informational draft, but I guess that's fine.